### ISSUE TITLE
Dev UI: Troubleshoot page gets populate for running workflows

### ISSUE THREAD

**Issue by sitapriyamoorthi** [FredHutch/shiny-cromwell#182]
Dont know if this is intended but when I go to the troubleshoot page for a workflow that is running fine i get a populated page
![Screenshot 2024-11-04 at 10 27 07 AM](https://github.com/user-attachments/assets/b68e0e58-8053-45c1-a96c-de289e9d6203)

**Comment by sckott** [comment]
yes, that's intended. removes the need to click a button. the intention is to revamp the output on the troubleshoot page to make everything clearer, but that hasn't happened yet
